server/controllers/dummyController.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response } from 'express';
+
+const dummyController = (req: Request, res: Response): Response<any, Record<string, any>> => (
+  res.json({ msg: 'The app is running correctly!' })
+);
+
+export default dummyController;
```

Comment:
big piece of advice
FUCK default exports :)
`export const dummyController`
the thing is, your code editor understands named exports better (because their names dont change or are defined by default) so when you wanna import them
you dont think on where you wanna import

you just write `dummyContr` and then results will appear asking you to auto import *work smart not hard*

Reply:
I like the concept!